### ISSUE TITLE
Closing 1st Window causes Program Exit when Multiple Windows are Open

### ISSUE THREAD

**Issue by hairycactus** [pdfarranger/pdfarranger#592]
### Observed Behaviour
When a 2nd/3rd/4th/etc window is opened (via `☰ > New Window (CTRL-N)`), closing the 1st window (whether empty or with loaded document) causes PDF Arranger to exit abruptly.

Note: Closing the window refers to either clicking the `X` on the titlebar, or using `☰ > Quit (CTRL-Q)`. Document refers to any PDF or supported image files.

This abrupt program exit occurs, even when document changes in the 2nd/3rd/4th/etc windows *have not been saved*. And there is no warning about unsaved changes in the 2nd/3rd/4th/etc windows.

If multiple windows (3 or more) are open, *neither* of the following actions will cause PDF Arranger to exit:-

- Closing additional windows (in any sequence), but leaving the *1st window open*  
- Closing any intermediate window, but leaving the *1st & last-opened windows open*

In other words, regardless of the number of open windows, only closing the 1st window will trigger a program exit. It's as if PDF Arranger is treating all the other open windows as non-existent.

### Expected Behaviour
Closing the 1st window (or any open window) should not result in the entire program exiting abruptly, especially when there are unsaved document changes in any of the windows.

A window should close only if user *chooses* to close it.

**Tested:** v1.8.0 (13 Nov 2021) portable, v1.7.1 (18 Mar 2021) portable from Github Releases

**OS:** Win 10 x64

Thanks !

**Comment by kbengs** [comment]
I am not able to reproduce it...
Could you try the version found here:
https://github.com/pdfarranger/pdfarranger/actions/runs/1531021604
Then start it from terminal and see if anything interesting is printed in terminal?

**Comment by dreua** [comment]
> I am not able to reproduce it...

Neither am I. Must be something very specific to @hairycactus 's Windows, no? If anyone else can reproduce or has some idea, please give us input.

**Comment by dreua** [comment]
Closing as there has been no response to kbengs request for more information.
